src/condor_utils/file_transfer.cpp

```diff
@@ -4046,14 +3990,15 @@ FileTransfer::InvokeMultiUploadPlugin(
 	CondorError &err,
 	long long &upload_bytes)
 {
-	auto result = InvokeMultipleFileTransferPlugin(err, exit_code, plugin, input, LocalProxyName.c_str(), true);
+	std::vector<ClassAd> resultAds;
+	auto result = InvokeMultipleFileTransferPlugin(err, exit_code, plugin, input, resultAds, LocalProxyName.c_str(), true);
 
 	// TODO: use plugin.name instead ?
 	const char * label = plugin.path.c_str();
 
 	int count = 0;
 	bool classad_contents_good = true;
-	for (const auto & xfer_result: pluginResultList) {
+	for (const auto & xfer_result: resultAds) {
```

Comment:
This looks broken: `resultAds` does not always contain every plug-in result ad, so isn't the protocol going to drift out of sync?